src/app/api/api.py

```diff
@@ -23,28 +23,62 @@
 
 router = APIRouter()
 
+QUALITY_MATRIX_DESCRIPTION = """Calculation of the quality matrix.
+    For each replication source and each property, e.g., `cm:creator`, the quality matrix returns the ratio of
+    elements which miss this entry compared to the total number of entries.
+    A missing entry may be `cm:creator = null`."""
+TAG_STATISTICS = "Statistics"
+
 
 @router.get(
     "/quality_matrix",
     status_code=HTTP_200_OK,
     response_model=List[ColumnOutputModel],
     responses={HTTP_404_NOT_FOUND: {"description": "Quality matrix not determinable"}},
-    tags=["Statistics"],
-    description="""Calculation of the quality matrix.
-    For each replication source and each property, e.g., `cm:creator`, the quality matrix returns the ratio of
-    elements which miss this entry compared to the total number of entries.
-    A missing entry may be `cm:creator = null`.""",
+    tags=[TAG_STATISTICS],
+    description=QUALITY_MATRIX_DESCRIPTION,
+)
+async def get_quality_matrix(timestamp: Optional[int] = None):
```

Comment:
I assume this optional timestamp can be passed as query parameter (i.e. `.../quality_matrix?timestamp=XYZ`). If so, then we don't need the other endpoint. But instead add the query parameter documentation to the  docstring,

Reply:
The first example here should also describe how to document a query parameter in the openapi spec!

https://fastapi.tiangolo.com/release-notes/?h=query#features